src/modules/roc_netio/target_posix/roc_netio/sendto.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include "roc_netio/sendto.h"
+#include "roc_core/errno_to_str.h"
+#include "roc_core/log.h"
+#include <sys/socket.h>
+#include <sys/types.h>
```

Comment:
Nitpick: please add an empty line between roc and system include blocks.

Reply:
Maybe it would be a good idea to add this, the preference for C-style casts... and other formatting decisions to the contributing guidelines (even adding a `clang-format` rule).

Reply:
Agree.Not sure about clang-format rule though, e.g. sometimes we have to use reinterpret_cast to suppress warnings.

Code style is (partially) documented here: https://roc-project.github.io/roc/docs/development/coding_guidelines.html